x/ibc/04-channel/types/channel.go

```diff
@@ -0,0 +1,67 @@
+package types
+
+// ChannelOrder defines if a channel is ORDERED or UNORDERED
+type ChannelOrder byte
+
+// channel order types
+const (
+	UNORDERED ChannelOrder = iota // packets can be delivered in any order, which may differ from the order in which they were sent.
```

Comment:
The 0-value should be `NONE` (so subsequent values are 1-indexed)

Reply:
what about  
```go
UNORDERED ChannelOrder = iota + 1
```

Reply:
That should work as well.